src/bindings/js/node/tests/unit/core.test.js

```diff
@@ -70,3 +71,32 @@ describe('ov.Core tests', () => {
     );
   });
 });
+
+describe('Core.query_model()', () => {
+  const core = new ov.Core();
+  const model = core.readModelSync(getModelPath().xml)
+  it('should throw an error in case of no arguments', () => {
+    assert.throws(
+      () => core.queryModel().then(),
+      /'queryModel' method called with incorrect parameters./
+    )
+  });
+  it('should throw an error in case of less arguments', () => {
+    assert.throws(
+      () => core.queryModel("Unexpected Argument").then(),
+      /'queryModel' method called with incorrect parameters./
+    )
+  });
+  it('should throw an error in case of incorrect arguments', () => {
+    assert.throws(
+      () => core.queryModel(model, "arg1", "arg2").then(),
+      /'queryModel' method called with incorrect parameters./
+    )
+  });
+
+  it('should have device returned as one of the properties', () => {
+    device = 'CPU'
+    query_model = core.queryModel(model, device)
+    assert(Object.values(query_model).includes(device));
```

Comment:
```suggestion
 });

  describe('Core.query_model()', () => {
    const core = new ov.Core();
    const model = core.readModelSync(getModelPath().xml);
    it('should throw an error in case of no arguments', () => {
      assert.throws(
        () => core.queryModel().then(),
        /'queryModel' method called with incorrect parameters./
      );
    });
    it('should throw an error in case of less arguments', () => {
      assert.throws(
        () => core.queryModel('Unexpected Argument').then(),
        /'queryModel' method called with incorrect parameters./
      );
    });
    it('should throw an error in case of incorrect arguments', () => {
      assert.throws(
        () => core.queryModel(model, 'arg1', 'arg2').then(),
        /'queryModel' method called with incorrect parameters./
      );
    });

    it('should have device returned as one of the properties', () => {
      const device = 'CPU';
      const queryModel = core.queryModel(model, device);
      assert(Object.values(queryModel).includes(device));
    });
  });
```
Major change: `describe('Core.query_model()'` is part of `describe('ov.Core tests'`
Plus formatting changes like missing semicolons, snakecase, missing const.

Reply:
Done. Please check once.